From 1f1d2dbd168aa6a589dcf9d100fc2bc1430c5e63 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Fri, 30 Sep 2011 03:31:03 +0200
Subject: [PATCH 2/2] disable musb0 on the bone

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 drivers/usb/musb/ti81xx.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/musb/ti81xx.c b/drivers/usb/musb/ti81xx.c
index 0625432..7208dc5 100644
--- a/drivers/usb/musb/ti81xx.c
+++ b/drivers/usb/musb/ti81xx.c
@@ -1022,6 +1022,9 @@ int ti81xx_musb_init(struct musb *musb)
 	u32 rev;
 	u8 mode;
 
+	if (musb->id == 0)
+		return -ENODEV;
+
 	if (musb->id < 2)
 		gmusb[musb->id] = musb;
 
-- 
1.6.6.1

